content/800-data-platform/200-projects/index.mdx

```diff
@@ -0,0 +1,80 @@
+3. [Import an existing application with a new database]()
+    * **Database**: provide a connection string to an empty database
+    * **GitHub repository**: select an existing GitHub repository
+    * **Use case**: Import your application and use an empty database to explore and test project creation and the features in the Prisma Data Platform.
+
+You can also edit your project settings (name and URL) and delete your project if you no longer need it.
+
+* [Edit project settings](/data-platform/projects/edit-settings)
+* [Delete a project](/data-platform/projects/delete-project)
+
+</TopBlock>
+
+## Prerequisites and considerations
+
+* [Sign up for the Prisma Data Platform](data-platform/create-manage-projects#sign-up-for-the-prisma-data-platform)
+* [Grant read and write permissions to your GitHub repositories](/data-platform/create-manage-projects#grant-read-and-write-permissions-to-the-prisma-github-app)
+
+### Sign up for the Prisma Data Platform
+
+To create projects in the Prisma Data Platform, you must sign up first. For more information see [Sign up for the Prisma Data Platform](/data-platform/accounts#sign-up-for-the-prisma-data-platform).
+
+### Grant read and write permissions to the Prisma GitHub app
+
+When you sign up for the Prisma Data Platform, you must grant access to the Prisma GitHub app so that it can read data about your GitHub profile and use that to log you in to the platform.
+
+In addition, you must also grant read and write permissions to the GitHub app so that it can read and create new repositories.
+
+#### Steps
+
+1. After you [sign up](/data-platform/accounts#sign-up-for-the-prisma-data-platform), go to the [_Projects_](https://cloud.prisma.io/projects) page.
+2. Click **New Project**.
+2. Click the **GitHub Accounts & Organizations** drop-down menu.
+3. If you do not see any GitHub accounts or organizations in the drop-down, select **+ Add an Organization or Account**.
+
+    <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1666309163/docs/pdp/create-manage-projects/grant-repo-permissions-prisma-github-01-click-github-org-accounts_n0ib1g.png"
+        alt="Prisma Data Platform - Grant read and write permissions to Prisma GitHub app  - Step 01 - Click the Add orgs and accounts drop-down"
+        width="750px"
+    />
+        
+4. On the new browser tab that opens, click **Install** to grant permissions to the Prisma GitHub app read and write permissions to your repositories.
+
+    <Admonition>
+
+    **Note**<br /><br />
+    The Prisma Data Platform requires read and write permissions to GitHub repositories to read your Prisma schema file and to create new repositories for new projects.
+
+    </Admonition>
+
+#### Result
+
+The Prisma GitHub app now has the required permissions to read the Prisma schema from the repositories you add for your projects in the Prisma Data Platform.
```

Comment:
I'm not sure quite what to suggest here, but I think this needs a rewrite. Don't we have a schema for each project, so shouldn't "schema" be plural? Maybe something like:

```suggestion
The Prisma GitHub app has the permissions that it needs to read your Prisma schemas and create new repositories for your Prisma Data Platform projects.
```